sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -3413,6 +3413,14 @@ object SQLConf {
     .booleanConf
     .createWithDefault(true)
 
+  val AVRO_STABLE_ID_FOR_UNION_TYPE = buildConf(
+    "spark.sql.avro.enableStableIdentifiersForUnionType")
+    .doc("When Avro is desrialized to SQL schema, the union type is converted to structure in a " +
+      "way that field names of the structure are stable with the type, in most cases.")
```

Comment:
The description seems to need revisions to clarify what is the difference between `true` and `false`.

Reply:
Especially, please mention the `case-sensitivity` cases.